x/leverage/keeper/keeper.go

```diff
@@ -389,22 +367,13 @@ func (k Keeper) Liquidate(
 	}
 
 	if directLiquidation {
-		// burn the uToken reward from borrower's collateral
-		if err = k.burnCollateral(ctx, borrowerAddr, collateralLiquidate); err != nil {
-			return sdk.Coin{}, sdk.Coin{}, sdk.Coin{}, err
-		}
-
-		// send base rewards from module to liquidator's account
-		if err = k.bankKeeper.SendCoinsFromModuleToAccount(
-			ctx, types.ModuleName, liquidatorAddr, sdk.NewCoins(baseReward),
-		); err != nil {
-			return sdk.Coin{}, sdk.Coin{}, sdk.Coin{}, err
-		}
+		err = k.redeemCollateral(ctx, borrowerAddr, liquidatorAddr, collateralLiquidate)
```

Comment:
Notice how `baseReward` from `getLiquidationAmounts` is no longer being used - instead it would presumably be recalculated in `redeemCollateral` (though the proposed `redeemCollateral` does not attempt to convert to base tokens.) This is a `uToken` reward, not a `directLiquidation` (see the if statement).

Reply:
right, `collateralLiquidate != baseReward`. I will undo this change.